google_analytics/templatetags/google_analytics_tags.py

```diff
@@ -55,10 +56,11 @@ def render(self, context):
         # append the debug parameter if requested
         if self.debug:
             params['utmdebug'] = 1
+
         # build and return the url
-        url = reverse('google-analytics')
-        if len(params) > 0:
-            url += '?' + urllib.urlencode(params)
+        ga_params = build_ga_params(request)
```

Comment:
@codiebeulaine you basically need a new tag that just does this bit and return the url